### ISSUE TITLE
Stuck In Detecting Hardware

### ISSUE THREAD

**Issue by Dighofx86** [BlissOS/bug_reports#4]
I tried version 16.8 of bliss os but got stuck on detecting hardware after selecting the installation menu.

I Try in Asus A455L
Intel Core i3 5005
Intel UHD Graphic 5005
Ram 4GB

**Comment by emiron98** [comment]
same on my Lenovo Ideapad 520IKB-80YL. I can boot the live version but trying to install it results in getting stuck on "detecting hardware" 
Motherboard model number: LNVNB161216
cpu: i7-7500u
Graphics : intel hd graphics 620, nvidia gt 940mx (will not be used)
Ram: 16 gb
EDIT: I was able to install Bliss os 16. I went into bios and disabled discrete graphics, intel sgx and rst premium. after the installation i reverted back to original settings. now it boots normally
